.github/workflows/graph.yml

```
@@ -0,0 +1,35 @@
+name: Deploy Graph
+
+on:
+  push:
+    branches: master
```

```suggestion
    branches: main
```
would also add workflow-dispatch to make it easier for us to test it in this PR